src/screens/TreeDetailsScreen.tsx

```diff
@@ -35,13 +41,19 @@ export default function TreeDetailsScreen({
     uuid: '',
     location: null,
     planted: null,
+    comments: [],
   });
 
   const [isEditing, setIsEditing] = useState<boolean>(false);
   const toggleEditing = useCallback(() => {
     setIsEditing(!isEditing);
   }, [isEditing]);
 
+  const [comment, setComment] = useState<Comment>({
+    uuid: '',
```

Comment:
how does assigning comments an ID work? is that designed so that they show up in chronological order and if so, how do we view the comments on a tree